packages/liveblocks-client/.eslintrc.js

```diff
@@ -0,0 +1,56 @@
+module.exports = {
+  root: true,
+  parser: "@typescript-eslint/parser",
+  parserOptions: { project: ["./tsconfig.json"] },
+  plugins: ["@typescript-eslint"],
+  extends: ["eslint:recommended", "plugin:@typescript-eslint/recommended"],
+  rules: {
+    // ----------------------------------------------------------------------
+    // NOTE: Only temporarily turned off!
+    // These checks are still GOOD IDEAS to re-enable later on, but for right
+    // now they're too noisy to enable.
+    // ----------------------------------------------------------------------
+    "@typescript-eslint/ban-ts-comment": "off",
+    "@typescript-eslint/no-explicit-any": "off",
+    "@typescript-eslint/no-non-null-asserted-optional-chain": "off",
+    "@typescript-eslint/no-non-null-assertion": "off",
```

Comment:
These are currently giving errors, but I think they're good hints of things we may want to come back to later and fix, so I'm temporarily disabling them here for now.